README.md

```diff
@@ -1,24 +1,264 @@
-# README
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Modify this file to match your project and remove sections that don't apply.
 
-Things you may want to cover:
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-* Ruby version
+OPTIONAL SECTIONS:
+- FAQ
 
-* System dependencies
+After you're finished please remove all the comments and instructions!
+-->
 
-* Configuration
+<div align="center">
 
-* Database creation
+  <h3><b>README Blog</b></h3>
 
-* Database initialization
+</div>
 
-* How to run the test suite
+<!-- TABLE OF CONTENTS -->
 
-* Services (job queues, cache servers, search engines, etc.)
+# 📗 Table of Contents
 
-* Deployment instructions
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-* ...
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [blog] <a name="about-project"></a>
+
+> This is a blog app.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- Ruby
+- PostgreSQL
+- Rails
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
```

Comment:
- [ ] As requested by the previous reviewer, kindly add some key features of this project 
`e.g setup database models for the blog app`